src/main/java/org/folio/services/validator/engine/ValidationEngineServiceImpl.java

```diff
@@ -216,15 +211,15 @@ private void validatePasswordByRexExpRule(final String password,
     String okapiURL = headers.get(OKAPI_URL_HEADER);
     String remoteModuleUrl = okapiURL + rule.getImplementationReference();
 
-    Future<String> future = Future.future();
+    Promise<String> promise = Promise.promise();
     HttpClientRequest passwordValidationRequest = httpClient.postAbs(remoteModuleUrl, validationResponse -> {
```

Comment:
Deprecation changes due to upgrading to Vert.x 3.8.1 > 3.6.2
* old: HttpClient request/response methods with Handler<HttpClientResponse>
* new: WebClient request/response methods with Handler<AsyncResult<HttpClientResponse>>

https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md#version-271

Should replacing HttpClient by WebClient be part of MODPWD-28 or a separate issue?

Reply:
I would suggest separate issue for this

Reply:
created https://issues.folio.org/browse/MODPWD-29